javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
+var user = []; //inline coment
+
+/**
```

Comment:
Typo inside comment in token 'Number'. Possible corrections:number (57),member (2),numbers (0)